index.js

```diff
@@ -1,5 +1,31 @@
 'use strict';
 var chalk = require('chalk');
+var isSupported = (function() {
+	if (process.platform !== 'win32') {
+		return true;
+	}
+
+	var env = process.env;
+
+	if (env.APPVEYOR) {
```

Comment:
We can remove code about this, and discuss it in the future, the focus is on Windows 10.